### PR TITLE
네비게이션 적용

### LINKED ISSUE #4: 네비게이션 구조 설계
구조
Root
- Login
- BottomTab
  - Home
  - Record
  - Running
    - RunTab
      - MoyeRun
      - personal or private Run
  - Mission
  - Friend
- Running

TODO
스크린, 네이게이션 타입 정의
타입 링크, 네비게이션 생성

### PR DESCRIPTION
**Pull request by sjsjsj1246**
Root Navigation
- Home
- Record
- Running
- Mission
- Friend

KakaoTalk_Photo_2021-09-22-21-19-09

/types.ts
/navigation

### CODE REVIEW

File: src/navigation/LinkingConfiguration.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { LinkingOptions } from "@react-navigation/native";
+import * as Linking from "expo-linking";
+
+import { RootStackParamList } from "../types";
+
+const linking: LinkingOptions<RootStackParamList> = {
```
Comment (Sangjun-man) [inline src/navigation/LinkingConfiguration.ts:6]: 네비게이션 링크 확인요

File: src/navigation/index.tsx [hunk 2]
```diff
@@ -71,49 +80,52 @@ function BottomTabNavigator() {
...
+        name="Friend"
+        component={FriendScreen}
+        options={{
+          title: "친구",
+          tabBarIcon: ({ color }) => <FriendIcon color={color} />,
         }}
       />
     </BottomTab.Navigator>
   );
 }
```
Comment (Sangjun-man) [inline src/navigation/index.tsx:131]: 화긴화긴

**Review by Sangjun-man — APPROVED** [review]
확인확인!!

File: src/navigation/index.tsx (hunk 2 above)
Comment (sjsjsj1246) [reply src/navigation/index.tsx:131]: 굿굿

**Pull request merged**